### PR TITLE
OCR2DR - Admin killswitch

### PR DESCRIPTION
**Pull request by justinkaseman**
### Description
Extends the OCR2DR Registry with OpenZeppelin's Pausable contract.
This allows the Registry owner to pause function in case of catastrophic failure.

In the paused state the following functions will only be callable by the Registry owner:
- cancelSubscription
- addConsumer
- removeConsumer
- acceptSubscriptionOwnerTransfer
- createSubscription
- oracleWithdraw
- startBilling
- fulfillAndBill

### CODE REVIEW

File: contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol [hunk 1]
```diff
@@ -718,6 +759,40 @@ contract OCR2DRRegistry is ConfirmedOwner, OCR2DRRegistryInterface, ERC677Receiv
     return false;
   }
 
+  /**
+   * @notice Time out all expired requests: unlocks funds and removes the ability for the request to be fulfilled
+   * @param subscriptionId - ID of the subscription
+   */
+
+  function timeoutRequests(uint64 subscriptionId) external onlySubOwner(subscriptionId) onlyActiveRegistry {
```
Comment (bolekk) [inline contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol:767]: Why is this part of this PR?

**Review by bolekk — COMMENTED** [review]
I'm a little confused by this PR - it seems to contain several unrelated changes. Is it still a draft or ready to review?

**Comment by justinkaseman** [comment]
> I think we need to consider implementing this https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol

That sounds good to me, I'll implement

**Review by KuphJr — CHANGES_REQUESTED** [review]
Where is the external function allowing the owner to `pause` and `unpause`? I only see the internal functions `_pause()` and `_unpause`?

**Comment by justinkaseman** [comment]
> Where is the external function allowing the owner to `pause` and `unpause`? I only see the internal functions `_pause()` and `_unpause`?

https://github.com/smartcontractkit/chainlink/pull/8121/files#diff-27f1b21cbbc1c90bc638ee9175f21a0bab8f511c5544b261dbe76c0e57b47553R89

**Review by KuphJr — DISMISSED** [review]

**Review by bolekk — DISMISSED** [review]
lgtm

**Review by KuphJr — APPROVED** [review]

**Review by pinebit — APPROVED** [review]

**Review by bolekk — APPROVED** [review]

**Pull request merged**